README.md

```diff
@@ -39,25 +39,25 @@ Create a file called `config.json` which has the following format.
 Fill the quotation marks with your desired namespaces, etc.
 (Note that `#` tokens won't be permitted in the actual JSON file.)
 
-* For a list of available locations, [see here](https://azure.microsoft.com/en-us/global-infrastructure/locations/).
-* For a list of available Linux Virtual Machines, [see here](https://azure.microsoft.com/en-gb/pricing/details/virtual-machines/linux/).
+* For a list of available data centre regions, [see here](https://azure.microsoft.com/en-us/global-infrastructure/locations/). This should be a _region_ and **not** a _location_, e.g. "West Europe" or "Central US". These can be equivalently written as `westeurope` and `centralus`, respectively.
+* For a list of available Linux Virtual Machines, [see here](https://docs.microsoft.com/en-gb/azure/virtual-machines/linux/sizes-general). This should be something like, e.g., `Standard_DS2_v3`.
 
 ```
 {
   "azure": {
     "subscription": "",  # Azure subscription name
     "res_grp_name": "",  # Azure Resource Group name
     "location": "",      # Azure Data Centre location
-    "node_count": "",    # Number of nodes to deploy
+    "node_count": 1,     # Number of nodes to deploy. 1 is the limit of a Free Trial subscription, 3 is preferrable.
```

Comment:
Actually, I think the limit for the Free subscription is 4 **cores**. Depending on what VM you choose, that will limit the number of nodes to one thing or another. For example, when I tried 3 nodes + Standard_D2s_v3, it resulted in 6 cores, and I got an error.

Reply:
Fun! Can we do 2 nodes + Standard_D2s_v3 = 4 cores to verify?

Reply:
I've re-run this:

1 node + Standard_D2s_v3 --> OK
2 nodes + Standard_D2s_v3 --> OK
3 nodes + Standard_D2s_v3 --> ERROR!! `Operation results in exceeding quota limits of Core. Maximum allowed: 4, Current in use: 4, Additional requested: 2.`

D2s has 2 cores, so I would say that the first 2 cases were 2 and 4 cores, respectively.

Reply:
Ok interesting! Thanks for doing that. What I might do then is leave this as 1 node in the example and update the README to reflect what we've learned here 🙂

Reply:
This is my subscription (Free Trial). And the list continues with the quotas in other services.
![image](https://user-images.githubusercontent.com/6095790/58251451-93ff7880-7d5b-11e9-941e-95a2de521f15.png)

Reply:
I added some discussion on the core quota and free trial subscriptions in general in [`75c2875`](https://github.com/alan-turing-institute/binderhub-deploy/pull/11/commits/75c2875612706d68f85429feb47212b45312e46b).